### PR TITLE
chore: normalize configuration values (develop)

### LINKED ISSUE #8666: Normalize default configuration
### Description

We need to normalize the default configuration values to match the values stored in `configuration/mainnet/*.properties`.

### Steps to reproduce

Not Applicable

### Additional context

_No response_

### Hedera network

other

### Version

Not Applicable

### Operating system

None

### PR DESCRIPTION
**Pull request by nathanklick**
### Related Issues 

- Closes #8666

### CODE REVIEW

**Review by jsync-swirlds — DISMISSED** [review]

**Review by poulok — APPROVED** [review]

**Review by Neeharika-Sompalli — DISMISSED** [review]

File: hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties [hunk 1]
```diff
@@ -38,15 +38,15 @@ files.throttleDefinitions=123
 hedera.firstUserEntity=1001
 hedera.realm=0
 hedera.shard=0
-ledger.id=0x03
+ledger.id=0x00
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties:41]: Is this intentional ?

File: hedera-node/configuration/mainnet/application.properties [hunk 2]
```diff
@@ -1 +1,12 @@
-
+autoRenew.targetTypes=
+contracts.knownBlockHash=055fac8eb788872957dfee45b7d40f7c94d66ea0a75ec703b993d57e441de574@33476932
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+sigs.expandFromLastSignedState=false
+staking.fees.stakingRewardPercentage=10
+staking.maxDailyStakeRewardThPerH=6_849
```
Comment (Neeharika-Sompalli) [inline hedera-node/configuration/mainnet/application.properties:8]: I think this property is no more in develop.

File: hedera-node/configuration/mainnet/application.properties [hunk 3]
```diff
@@ -1 +1,12 @@
...
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
+staking.rewardRate=48630136986000
```
Comment (Neeharika-Sompalli) [inline hedera-node/configuration/mainnet/application.properties:11]: same with this.

**Review by Neeharika-Sompalli — DISMISSED** [review]

**Review by jsync-swirlds — APPROVED** [review]
As this no longer matches the _current_ contents of file 121, we should definitely verify the following:
1. These changes are correct for future release to mainnet
2. Any changes to this file will be reflected in file 121 as part of future release processes.

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by poulok — APPROVED** [review]

**Comment by mhess-swl** [comment]
Force pushed to get correct DCO signoff

**Comment by jsync-swirlds** [comment]
> Force pushed to get correct DCO signoff

and now you somehow have the last 30 commits from develop incorporated in the PR...

**Comment by mhess-swl** [comment]
> > Force pushed to get correct DCO signoff
> 
> and now you somehow have the last 30 commits from develop copied and incorporated in the PR...

I also merged with latest develop

**Comment by jsync-swirlds** [comment]
> > > Force pushed to get correct DCO signoff
> > 
> > 
> > and now you somehow have the last 30 commits from develop copied and incorporated in the PR...
> 
> I also merged with latest develop

It did not merge correctly, a regular correct merge will not show the merged commits as new commits.

**Review by jsync-swirlds — APPROVED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM !

**Review by JeffreyDallas — APPROVED** [review]

**Pull request merged**